internal/service/keys/model_kerberos_keys.go

```diff
@@ -0,0 +1,75 @@
+package keys
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/keys"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type KerberosKeysModel struct {
+	Items types.List `tfsdk:"items"`
+}
+
+var KerberosKeysAttrTypes = map[string]attr.Type{
+	"items": types.ListType{ElemType: types.ObjectType{AttrTypes: KerberosKeyAttrTypes}},
+}
```

Comment:
is this model used anywhere, if not, it can be deleted

Reply:
updated

Reply:
you can delete the whole file actually

Reply:
removed the file. Unit test and other local test passed.